book/specifications/kimchi/Specification.toml

```diff
@@ -2,7 +2,7 @@
 name = "Kimchi"
 description = "This specification describes the Kimchi proof system."
 version = "0.1.0"
-authors = ["Izaak Meckler", "Vanishree Rao", "Matthew Ryan", "Joseph Spadavecchia", "David Wong", "Vitaly Zelov"]
+authors = ["Izaak Meckler", "Anaïs Querol", "Vanishree Rao", "Matthew Ryan", "Joseph Spadavecchia", "David Wong", "Vitaly Zelov"]
```

Comment:
can't believe you were not already there!

Reply:
It's an honour 😉